build/params_testnet.go

```diff
@@ -20,22 +20,21 @@ const BreezeGasTampingDuration = 120
 
 const UpgradeSmokeHeight = -1
 
-const UpgradeIgnitionHeight = 94000
+const UpgradeIgnitionHeight = -1
```

Comment:
This might actually need to be set to something.

Reply:
I've pushed another commit which will set this to `23950`.

Reply:
It _should_ be fine to set it to -1